File: python/lesson2/setaq1.py
values = []

print("Enter 5 integers:")

for i in range(5):
    num = int(input(f"Enter integer {i + 1}: "))
    values.append(num)
if len(values) != len(set(values)):
    print("DUPLICATES")
else:
    print("ALL UNIQUE")

File: lesson4/setaq1.java
import java.util.Scanner;

public class setaq1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter a list of values separated by spaces:");

        String input = scanner.nextLine();

        String[] inputs = input.split(" ");

        int count = 0;

        for (String arg : inputs) {
            try {
                Integer.parseInt(arg);
                count++;
            } catch (NumberFormatException e) {
            }
        }

        System.out.println("Number of integers in the given list: " + count);

        scanner.close();
    }
}
